shared/settings/advanced/index.tsx

```diff
@@ -235,6 +236,14 @@ class Developer extends React.Component<Props, State> {
             style={styles.developerButtons}
           />
         )}
+        {flags.admin && (
+          <Kb.Button
```

Comment:
This means never in TestFlight?

Reply:
Ah right, I guess that won't work. Hmm, I'm not sure what the best way to show this in the mobile app is then.

Reply:
We already have the hidden buttons behind the clicks

Reply:
K I guess that works, will do that.